distribution/lib/Standard/Base/0.0.0-dev/src/Runtime.enso

```diff
@@ -52,8 +53,20 @@ get_stack_trace =
 gc : Nothing
 gc = @Builtin_Method "Runtime.gc"
 
+## ADVANCED
+
+   Asserts that the given action succeeds, otherwise throws a panic.
+
+   Assertions are disable by default, meaning that call to this method is
+   a no-op. To enable assertions, set the environment variable
+   `ENSO_ENABLE_ASSERTIONS=true`
+assert : (Any -> Boolean) -> Text -> Nothing ! Assertion_Error
```

Comment:
I'm so happy about this!

Reply:
I would (slightly) prefer connecting this functionality with [java -ea](https://github.com/enso-org/enso/pull/7883#discussion_r1336629175).

Reply:
> I would (slightly) prefer connecting this functionality with [java -ea](https://github.com/enso-org/enso/pull/7883#discussion_r1336629175).

Done in 72cd8361cb337d34f7e917c0a6af18ea48e0e90b